apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.html.ts

```diff
@@ -321,17 +324,30 @@ export function InstructorAssessmentGroups({
   `.toString();
 }
 
-function AddMembersForm({ row, csrfToken }: { row: GroupUsersRow; csrfToken: string }) {
+function AddMembersForm({
+  row,
+  uidsLimit,
+  csrfToken,
+}: {
+  row: GroupUsersRow;
+  uidsLimit: number;
+  csrfToken: string;
+}) {
   return html`
     <form name="add-member-form" method="POST">
-      UIDs:
-      <input
-        type="text"
-        class="form-control"
-        placeholder="A@ex.com, B@ex.com"
-        name="add_member_uids"
-      />
-      <br />
+      <div class="form-group">
+        <label for="add_member_uids">UIDs</label>
+        <input
+          type="text"
+          class="form-control"
+          placeholder="student@example.com"
```

Comment:
Did we end up going with `.com` or `.edu` for this?

Reply:
`.com`, because https://www.rfc-editor.org/rfc/rfc2606.html doesn't explicitly include `example.edu`.